### DISCUSSION TITLE
Pytorch Lightning Flash not working on Colab

### DISCUSSION THREAD

**Discussion by RafayF1** [Lightning-AI/pytorch-lightning#18915] [General]
Hi, I'll get straight to the point. Is Pytorch Lightning Flash completely depreciated or can we still use the model and datamodule classes? In colab, I am encountering dependencies errors even though I have pip installed it and the imports commands are successfully executed. Hopefully, someone will let me know ASAP. Thank you.

**Comment by V0XNIHILI** [comment]
Do you know why PyTorch lightning is archived?
